README.md

```diff
@@ -1,205 +1,82 @@
 <p align="center">
-  <img src="assets/logo.svg" alt="Meilisearch" width="200" height="200" />
+  <img width="350" src="assets/meilisearch-logo-light.svg?sanitize=true#gh-light-mode-only">
+  <img width="350" src="assets/meilisearch-logo-dark.svg?sanitize=true#gh-dark-mode-only">
 </p>
 
-<h1 align="center">Meilisearch</h1>
-
 <h4 align="center">
   <a href="https://www.meilisearch.com">Website</a> |
   <a href="https://roadmap.meilisearch.com/tabs/1-under-consideration">Roadmap</a> |
   <a href="https://blog.meilisearch.com">Blog</a> |
-  <a href="https://fr.linkedin.com/company/meilisearch">LinkedIn</a> |
-  <a href="https://twitter.com/meilisearch">Twitter</a> |
   <a href="https://docs.meilisearch.com">Documentation</a> |
-  <a href="https://docs.meilisearch.com/faq/">FAQ</a>
+  <a href="https://docs.meilisearch.com/faq/">FAQ</a> |
+  <a href="https://slack.meilisearch.com">Slack</a> |
 </h4>
 
 <p align="center">
   <a href="https://github.com/meilisearch/meilisearch/actions"><img src="https://github.com/meilisearch/meilisearch/workflows/Cargo%20test/badge.svg" alt="Build Status"></a>
   <a href="https://deps.rs/repo/github/meilisearch/meilisearch"><img src="https://deps.rs/repo/github/meilisearch/meilisearch/status.svg" alt="Dependency status"></a>
   <a href="https://github.com/meilisearch/meilisearch/blob/main/LICENSE"><img src="https://img.shields.io/badge/license-MIT-informational" alt="License"></a>
-  <a href="https://slack.meilisearch.com"><img src="https://img.shields.io/badge/slack-meilisearch-blue.svg?logo=slack" alt="Slack"></a>
-  <a href="https://github.com/meilisearch/meilisearch/discussions" alt="Discussions"><img src="https://img.shields.io/badge/github-discussions-red" /></a>
   <a href="https://app.bors.tech/repositories/26457"><img src="https://bors.tech/images/badge_small.svg" alt="Bors enabled"></a>
 </p>
 
-<p align="center">⚡ Lightning Fast, Ultra Relevant, and Typo-Tolerant Search Engine 🔍</p>
+<p align="center">⚡ An open-source, lightning-fast, and hyper-relevant search engine that fits effortlessly into your apps, websites, and workflow 🔍</p>
```

Comment:
For me it's a little bit too long @gmourier 
WDYT people?

Reply:
The "hyper-relevant" is maybe too much.

Reply:
WDYT @gmourier?

Reply:
I think that open source is a little bit obvious in a Github repository. I like things like `A lightning-fast search engine that fits effortlessly into your apps, websites, and workflow`.